schemas/external/repo/directory.schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id": "http://ns.adobe.com/adobecloud/core/1.0/directory",
+  "type": "object",
+  "title": "Directory",
+  "description": "A directory in Adobe Cloud Platform.",
+  "meta:extends": [
+    "http://ns.adobe.com/adobecloud/core/1.0#/definitions/common-properties",
+    "http://ns.adobe.com/adobecloud/core/1.0#/definitions/date-properties",
+    "https://ns.adobe.com/xdm/external/hal/resource#/definitions/hal"
+  ],
+  "definitions": {
+    "directory": {
+      "properties": {
+        "dc:format": {
+          "type": "string",
+          "title": "Format",
+          "const": "application/vnd.adobecloud.directory+json"
+        },
+        "children": {
```

Comment:
Shouldn't this belong to HAL _links? Or to _embedded if simple children representation needs to be included?

Reply:
Not according to [ACP spec](https://git.corp.adobe.com/pages/AdobeCloudPlatform/api-spec/repository_api/hypermedia.html), which I just tried to model here.